testnets/outbedevnet/chain.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chain.schema.json",
```

Comment:
The schema locations are required to be relative locations (so like ../../chain.schema.json)